src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -2629,7 +2669,15 @@ SpellMissInfo Unit::MagicSpellHitResult(Unit* victim, SpellInfo const* spellInfo
         }
 
         if (hasAura)
-            tmp += victim->GetTotalAuraModifierByMiscValue(SPELL_AURA_MOD_DEBUFF_RESISTANCE, static_cast<int32>(spellInfo->Dispel)) * 100;
```

Comment:
GetTotalAuraModifier* methods already take into account stack rules present in DB